EDDI/ChangeLog.md

```diff
@@ -3,8 +3,12 @@
 ### 3.0.0-rc2
   * Core
     * The EDSM responder has been updated to send data to EDSM per their revised API. 
+    * Incorporated new material transaction events
+    * EDDI's Material Monitor will now auto-calculate maximum material limits when not otherwise defined... provided the material rarity is known.
   * Speech Responder
-    * Added 'Fighter rebuilt' event
+      * Added `Fighter rebuilt` event
+      * Added `Material trade` event
```

Comment:
Are these event names truly `code voice` for Voice Attack users? For standalone users they are just names in the Speech Responder tab.

Reply:
Define `code voice`?

Reply:
The corresponding VoiceAttack commands shall be `((EDDI material traded))` and `((EDDI technology broker))` with auto-generated variables (as described at https://github.com/EDCD/EDDI/wiki/VoiceAttack-Integration#running-commands-on-eddi-events)